### DISCUSSION TITLE
Remove From Array

### DISCUSSION THREAD

**Discussion by SupertigerDev** [prisma/prisma#15460] [Q&A] [answered]
if i got a prisma field like this `roleIds: String[]` How can I delete 1 value from many users?
```js
await prisma.serverMember.updateMany({where: {roleIds: {has: roleId}}, data: {
  roleIds: // what goes here?
}});
```

**Comment by austincrim** [comment, marked as answer]
Hey @Supertigerr 👋,

You'll need pass the list of `roleIds` with the value you want deleted excluded from the list.
